Source/engine/render/text_render.cpp

```diff
@@ -416,9 +416,13 @@ uint32_t DoDrawString(const Surface &out, std::string_view text, Rectangle rect,
     TextRenderOptions &opts)
 {
 	CurrentFont currentFont;
-	int curSpacing = HasAnyOf(opts.flags, UiFlags::KerningFitSpacing)
-	    ? AdjustSpacingToFitHorizontally(lineWidth, opts.spacing, charactersInLine, rect.size.width)
-	    : opts.spacing;
+	int curSpacing = opts.spacing;
+	if (HasAnyOf(opts.flags, UiFlags::KerningFitSpacing)) {

```

Comment:
There is very similar code in `DrawStringWithColors` that also needs to be adjusted in the same way (search the file for `int curSpacing = HasAnyOf`)

Reply:
Here:
![image](https://github.com/user-attachments/assets/392ca198-49bf-44bb-a281-2f1d6d016ba7)